Build /cloud/private-cloud page

minkyngkm opened the pull request:
## Done

- Build `/cloud/private-cloud` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cloud/private-cloud
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/19g24YRYzF5dBpUKI1QzTG7fXKj1z0OvwgFll69dFoOs/edit#) and [Design](https://app.zeplin.io/project/6141d0b4743dc928767d815e/screen/6141d0e6a12a9c2971df3d0a).
- [Demo](https://ubuntu-com-10437.demos.haus/cloud/private-cloud)

## Issue / Card

Fixes[ #4410](https://github.com/canonical-web-and-design/web-squad/issues/4410)

## Screenshots
<img width="1440" alt="What is private cloud? | Ubuntu ()" src="https://user-images.githubusercontent.com/57550290/134302272-9091b1f7-d612-4f9a-a2b6-e309c1ef4714.png">

webteam-app commented:
Demo starting at https://ubuntu-com-10437.demos.haus

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+  <div class="row">
+    <div class="col-8">
+			<h1>What is private cloud?</h1>
+			<p>Private cloud is a cloud computing environment that is exclusively dedicated to a single entity or a service. It runs on the organisation's premises or in an external data centre. It is managed by the organisation’s operations team or a managed service provider.</p>
```
The indentation looks off here, and:

bethcollins92 replied:
```suggestion
			<p>Private cloud is a cloud computing environment that is exclusively dedicated to a single entity or a service. It runs on the organisation's premises or in an external data centre. It is managed by the organisation's operations team or a managed service provider.</p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+  <div class="row u-equal-height">
+    <div class="col-4">
+			<h2>How does a private cloud work?</h2>
+			<p>Like other cloud architectures, a private cloud enables on-demand resource provisioning through a self-service portal. This improves developer’s agility and simplifies the data centre management process.</p>
```
```suggestion
			<p>Like other cloud architectures, a private cloud enables on-demand resource provisioning through a self-service portal. This improves developer's agility and simplifies the data centre management process.</p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+    </div>
+    <div class="col-8">
+      <p>While private cloud is a single-tenant environment where all resources are available to a single entity or a service, public cloud is a multi-tenant environment where all resources are shared among various tenants (various organisations, services or end users) who typically do not know each other at all.</p>
+			<p>Another significant difference is in the <a href="/engage/cloud-pricing-report-2021">total cost of ownership (TCO)</a>. As public clouds implement so-called pay-as-you-go (PAYG) billing, they tend to be more economical in the short term and on a small scale. However, once the number of workloads continues to grow, their costs grow too, leading to an inflation of TCO. Thus, using a <a href=" ...
```
The hybrid cloud link points to the new page which doesn't exist yet. If you want to merge this before you'll have to remove this link for now and then make an issue to remind you to add it

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+				}}
+				</div>
+				<h4>Increased performance</h4>
+				<p class="u-sv3">As private clouds usually run on an organisation’s premises and there is no resource sharing, they usually provide better performance compared to public cloud infrastructure.</p>
```
```suggestion
				<p class="u-sv3">As private clouds usually run on an organisation's premises and there is no resource sharing, they usually provide better performance compared to public cloud infrastructure.</p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+    </div>
+    <div class="col-8">
+			<p>While public clouds provide an immediate access to the infrastructure-as-a-service (IaaS) offering with no upfront costs, their TCO can grow very fast depending on the number of workloads and the timeframe.</p>
+			<p>On the other hand, private clouds prove to be more cost-effective when running workloads in the long term and at scale, but they require significant upfront investments. Therefore, using a <a href="/cloud/hybrid-cloud">hybrid cloud architecture</a> is usually the way to achieve infrastructure cost optimisation.</p>
```
Same again with this hybrid cloud link

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+					<li class="p-list__item u-sv1"><a href="/telco">Telco NFV&nbsp;&rsaquo;</a></li>
+					<li class="p-list__item u-sv1"><a href="/server/docs/databases-introduction">Databases&nbsp;&rsaquo;</a></li>
+					<li class="p-list__item u-sv1"><a href="/kubeflow">AI/ML&nbsp;&rsaquo;</a></li>
+					<li class="p-list__item u-sv1"><a hef="/server/docs/web-servers-introduction">Web applications&nbsp;&rsaquo;</a></li>
```
```suggestion
					<li class="p-list__item u-sv1"><a href="/server/docs/web-servers-introduction">Web applications&nbsp;&rsaquo;</a></li>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+          }}
+					</div>
+					<h4>On-prem</h4>
+					<p class="u-sv3">Private cloud - in this case the cloud is deployed on the organisation's premises, inside of their own data centre. The customer provides hardware and hosting facilities, and usually takes care of the private cloud maintenance and operations. On-prem private clouds are best for big enterprises who have existing physical infrastructure and enough human resources to manage the ...
```
```suggestion
					<p class="u-sv3">Private cloud &mdash; in this case the cloud is deployed on the organisation's premises, inside of their own data centre. The customer provides hardware and hosting facilities, and usually takes care of the private cloud maintenance and operations. On-prem private clouds are best for big enterprises who have existing physical infrastructure and enough human resources to manage the cloud.</p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+				}}
+				</div>
+				<h4>Hosted</h4>
+				<p class="u-sv3">Private cloud - in this case the private cloud runs in a data centre that is outside the organisation’s facilities. The customer is still responsible for providing the hardware, but all hosting facilities (space, electricity, cooling, etc.) are delivered by the service provider. Hosted private clouds are best for organisations who do not have their own data centre and are not ...
```
```suggestion
				<p class="u-sv3">Private cloud &mdash; in this case the private cloud runs in a data centre that is outside the organisation’s facilities. The customer is still responsible for providing the hardware, but all hosting facilities (space, electricity, cooling, etc.) are delivered by the service provider. Hosted private clouds are best for organisations who do not have their own data centre and are not going to build one in the foreseeable future.</p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+          }}
+				</div>
+				<h4>Managed</h4>
+				<p>Private cloud - the private cloud is fully managed by a third party organisation (aka managed service provider). The customer provides the hardware, but cloud operations and maintenance tasks are outsourced. The cloud can either run on the organisation's premises or in the managed service provider data centre. Managed private clouds prove to be more economical for up to hundreds of nodes,  ...
```
```suggestion
				<p>Private cloud &mdash; the private cloud is fully managed by a third party organisation (aka managed service provider). The customer provides the hardware, but cloud operations and maintenance tasks are outsourced. The cloud can either run on the organisation's premises or in the managed service provider data centre. Managed private clouds prove to be more economical for up to hundreds of nodes, because they do not require hiring a dedicated operations team.</p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+				</div>
+				<h4>Hosted</h4>
+				<p class="u-sv3">Private cloud - in this case the private cloud runs in a data centre that is outside the organisation’s facilities. The customer is still responsible for providing the hardware, but all hosting facilities (space, electricity, cooling, etc.) are delivered by the service provider. Hosted private clouds are best for organisations who do not have their own data centre and are not ...
+				<p><a href="/openstack/what-is-openstack">More about management software&nbsp;&rsaquo;</a></p>
```
```suggestion
				<p><a href="/engage/private-cloud-cost-analysis-webinar">More about hosted cloud&nbsp;&rsaquo;</a></p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+				</div>
+				<h4>Managed</h4>
+				<p>Private cloud - the private cloud is fully managed by a third party organisation (aka managed service provider). The customer provides the hardware, but cloud operations and maintenance tasks are outsourced. The cloud can either run on the organisation's premises or in the managed service provider data centre. Managed private clouds prove to be more economical for up to hundreds of nodes,  ...
+				<p><a href="https://juju.is" class="p-link--external">More about automation</a></p>
```
```suggestion
				<p><a href="/openstack/managed">More about managed cloud</a></p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,483 @@
+		</div>
+	</div>
+</section>
+{% endblock content %}
```
```suggestion
{% endblock content %}

```

minkyngkm replied:
Would it mean insert a blank line before and after `{% endblock content %}`?

bethcollins92 replied:
Just after! It just complains if there isn't an empty line at the end that's all

minkyngkm replied:
Oh okay I added one before as well. Will remove it!

bethcollins92 reviewed (changes requested):
Looking great Min 🥇 - just a few tiny comments in the code and some incorrect links

minkyngkm commented:
Thanks @bethcollins92 for reviewing! I have fixed all you mentioned and also updated the images sizes and the sub-titles to `<h4>` following Zeplin. Hopefully it looks better! 
![image](https://user-images.githubusercontent.com/57550290/134368124-1678cc65-8248-4824-82eb-92a0e97f4859.png)

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,461 @@
+      </div>
+      <h4>Managed</h4>
+      <p>Private cloud &mdash; the private cloud is fully managed by a third party organisation (aka managed service provider). The customer provides the hardware, but cloud operations and maintenance tasks are outsourced. The cloud can either run on the organisation's premises or in the managed service provider data centre. Managed private clouds prove to be more economical for up to hundreds of ...
+      <p><a href="/openstack/managed">More about managed cloud</a></p>
```
```suggestion
      <p><a href="/openstack/managed">More about managed cloud&nbsp;&rsaquo;</a></p>
```

bethcollins92 commented on templates/cloud/private-cloud.html:
```diff
@@ -0,0 +1,461 @@
+      </div>
+      <h4>Hosted</h4>
+      <p class="u-sv3">Private cloud &mdash; in this case the private cloud runs in a data centre that is outside the organisation's facilities. The customer is still responsible for providing the hardware, but all hosting facilities (space, electricity, cooling, etc.) are delivered by the service provider. Hosted private clouds are best for organisations who do not have their own data centre and ...
+      <p><a href="/engage/private-cloud-cost-analysis-webinar">MMore about hosted cloud&nbsp;&rsaquo;</a></p>
```
```suggestion
      <p><a href="/engage/private-cloud-cost-analysis-webinar">More about hosted cloud&nbsp;&rsaquo;</a></p>
```

bethcollins92 reviewed (approved):
Almost there, I'll approve with the couple of changes suggested. Also please check the copy doc as I commented on one of the changes you must have missed.
